spec/annexes.html

```diff
@@ -108,6 +108,15 @@ <h1>Implementation Dependent Behaviour</h1>
         <li>
           Support for the Unicode extensions keys *"kf"*, *"kn"* and the parallel options properties *"caseFirst"*, *"numeric"* (<emu-xref href="#sec-Intl.Locale"></emu-xref>)
         </li>
+        <li>
+          The set of preferred calendars (<emu-xref href="#sec-calendars-of-locale"></emu-xref>), collations (<emu-xref href="#sec-collations-of-locale"></emu-xref>), hour cycles (<emu-xref href="#sec-hour-cycles-of-locale"></emu-xref>), numbering systems (<emu-xref href="#sec-numbering-systems-of-locale"></emu-xref>), and time zones (<emu-xref href="#sec-time-zones-of-locale"></emu-xref>) per locale
+        </li>
+        <li>
+          The default general ordering of characters within a line per locale (<emu-xref href="#sec-locale-direction"></emu-xref>)
+        </li>
+        <li>
+          The 'first' day of a week and which days of the week are considered as part of the 'weekend', for calendar purposes, per locale (<emu-xref href="#sec-week-info-of-locale"></emu-xref>)
```

Comment:
We don't use `'` as a quote in prose, and I don't think "first" and "weekend" need quoting:
```suggestion
          The first day of each week and which days of the week are considered as part of the weekend, for calendar purposes, per locale (<emu-xref href="#sec-week-info-of-locale"></emu-xref>)
```
But if you do want to keep quoting, the characters to use would be `“…”`:
```suggestion
          The “first” day of each week and which days of the week are considered as part of the “weekend”, for calendar purposes, per locale (<emu-xref href="#sec-week-info-of-locale"></emu-xref>)
```